plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/TestBigQueryConnectorTest.java

```diff
@@ -582,7 +583,20 @@ public void testSymbolAliasing()
         assertUpdate("DROP TABLE " + tableName);
     }
 
-    private void onBigQuery(String sql)
+    @Test
+    public void testSelectMaterializedView()
+    {
+        String materializedView = "test.test_materialized_view" + randomTableSuffix();
+        try {
+            onBigQuery("CREATE MATERIALIZED VIEW " + materializedView + " AS SELECT count(1) AS cnt FROM tpch.region");
+            assertQuery("SELECT * FROM " + materializedView, "VALUES 5");
+        }
+        finally {
+            onBigQuery("DROP MATERIALIZED VIEW IF EXISTS " + materializedView);
```

Comment:
Can we also test things like the schema (i.e. DESCRIBE) and some metadata operation like `DROP TABLE` (which I think would fail)?

Reply:
Added some assertions.